tests/AutoReview/TransformerTest.php

```diff
@@ -63,4 +64,40 @@ public function provideTransformerCases()
 
         return $transformersArray;
     }
+
+    /**
+     * @requires PHP 7.1
+     *
+     * @dataProvider provideTransformerPriorityCases()
+     */
+    public function testTransformerPriority(TransformerInterface $first, TransformerInterface $second)
+    {
+        $this->assertLessThan(
+            $first->getPriority(),
+            $second->getPriority(),
+            sprintf('"%s" should have less priority than "%s"', \get_class($second), \get_class($first))
+        );
+    }
+
+    public function provideTransformerPriorityCases()
+    {
+        if (\PHP_VERSION < 70100) {
```

Comment:
please drop it, as you already used `@requires PHP 7.1` annotation

Reply:
Sadly this is required, because there is something (might be phpunit itself) that calls dataproviders, even if they are unused. And because not all transformers are available, the "big list" would error, but I wanted to maintain the same format as for the fixer priority cases.

Reply:
then, why we don't need condition like that with all `@requires PHP x.y` we have in other places ?

Reply:
Because of the way the test currently gets the transformers, which I just reused. I can probably rewrite that.